pkg/pipelines/bootstrap.go

```diff
@@ -483,6 +487,7 @@ func createCICDResources(fs afero.Fs, repo scm.Repository, pipelineConfig *confi
 			return nil, err
 		}
 		outputs = res.Merge(outputs, trackerResources)
+		log.Success("Optional Pipelines tracker has been configured")
```

Comment:
We can drop "Optional" here, as it is implied by the fact that we asked the user if they wanted it (or they explicitly configured it).